aws/ci_build

```diff
@@ -67,7 +68,31 @@ def build
 
     RakeUtils.git_fetch
     count = RakeUtils.git_update_count
+
+    # Use a lock file to prevent git-pull and unison, both modify files in local
```

Comment:
Tiny NIT: `git-pull and unison, both modify files` -> `git-pull and unison, which both modify files`